index.html

```diff
@@ -10,6 +10,14 @@
     <script defer src="script.js"></script>
   </head>
   <body>
+    <ul>
+      <li><a class="active" href="index.html">Home</a></li>
+      <li><a href="updates.html">Updates</a></li>
+      <li style="float:right"><a href="https://hypixel.net/threads/hypixel-skyblock-goal-schema.3944822/" target="_blank" aria-label="Go to the Hypixel Forums Thread">Thread</a></li>
+      <li style="float:right"><a href="https://github.com/JavascriptLearner815/hypixel-skyblock-goal-schema/discussions/1" target="_blank" aria-label="Go to the Discussions Tab on GitHub">Discussions</a></li>
+      <li style="float:right"><a href="https://github.com/JavascriptLearner815/hypixel-skyblock-goal-schema/issues" target="_blank" aria-label="Go to the Issues Tab on GitHub">Issues</a></li>
+      <li style="float:right"><a href="https://github.com/JavascriptLearner815/hypixel-skyblock-goal-schema" target="_blank" aria-label="Go to the Code Tab on GitHub">Code</a></li>
+    </ul>
```

Comment:
Put this in between two `<hr>`s and in between the title and the updates section.

Also, add the version counter back here.